app/services/letter_sending_service.rb

```diff
@@ -37,6 +38,7 @@ def deliver! # rubocop:disable Metrics/AbcSize
     letter_record.notify_id = response.id
     letter_record.status = "accepted"
     letter_record.save!
+    update_audit("sent")
```

Comment:
possibly out of scope but should we put these multi record saves in a transaction?